modular_skyrat/modules/gun_dealer/code/gun_dealer.dm

```diff
@@ -0,0 +1,97 @@
+/datum/job/gun_dealer
+	title = "Gun Dealer"
+	description = "Sell guns, sell ammo, sell excuses for Security to arrest people."
+	department_head = list("Quartermaster")
+	job_spawn_title = "Cargo Technician"
+	faction = FACTION_STATION
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the Quartermaster"
+	selection_color = "#ffeeee"
+	exp_granted_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/gun_dealer
+	plasmaman_outfit = /datum/outfit/plasmaman/cargo
+
+	paycheck = PAYCHECK_MEDIUM
+	paycheck_department = ACCOUNT_CAR
+
+	display_order = JOB_DISPLAY_ORDER_CARGO_TECHNICIAN
+	bounty_types = CIV_JOB_RANDOM
+	departments_list = list(
+		/datum/job_department/cargo,
+		)
+
+	family_heirlooms = list(/obj/item/clipboard)
+
+	mail_goodies = list(
+		/obj/item/pizzabox = 10,
+		/obj/item/stack/sheet/mineral/gold = 5,
+		/obj/item/stack/sheet/mineral/uranium = 4,
+		/obj/item/stack/sheet/mineral/diamond = 3,
+		/obj/item/gun/ballistic/rifle/boltaction = 1
```

Comment:
Shouldn't this job receive ammo in the mail instead of the cargo tech's default list?
Maybe chuck the ammo pouch on the highest chance?

Reply:
Yeah, that's just a placeholder for now.